cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/DcLineSegmentConversion.java

```diff
@@ -30,30 +34,78 @@ public DcLineSegmentConversion(PropertyBag l, Context context) {
         cconverter2 = context.dc().cgmesConverterFor(iconverter2);
     }
 
+    private void updateConverterStations(double pAC) {
+
+        // poleLossP is the active power loss at a DC Pole
```

Comment:
Leave a more detailed comment:
        // poleLossP is the active power loss at a DC Pole
        // for lossless operation: P(DC) = P(AC) => lossFactor = 0
        // for rectifier operation (conversion from AC to DC) with losses: P(DC) = P(AC) - poleLossP 
        // In IIDM, for rectifier operation P(DC) / P(AC) = 100 - lossFactor
        // => P(DC) / P(AC) = 1 - poleLossP / P(AC) = 1 - lossFactor / 100
        // for inverter operation (conversion from DC to AC) with losses: P(DC) = P(AC) + poleLossP 
        // In IIDM, for inverter operation P(AC) / P(DC) = 100 - lossFactor / 100
        // => P(AC) / P(DC) = 1 - poleLossP / P(DC) = 1 - poleLossP / (P(AC) + poleLossP) = 100 - lossFactor / 100